docs/proposals/20191030-machine-health-checking.md

```diff
@@ -105,14 +104,12 @@ As an operator of a Management Cluster, I want my machines to be self-healing an
 
 ### Implementation Details/Notes/Constraints
 
-#### Machine annotation:
+#### Machine condition:
 ```go
-const MachineUnhealthyAnnotation = "machine.cluster.x-k8s.io/unhealthy"
+const ConditionHealthCheckPassed ConditionType = "HealthCheckPassed"
```

Comment:
```suggestion
const ConditionHealthCheckSucceeded ConditionType = "HealthCheckSucceeded"
```

IIRC we're trying to standardize on Ready/Succeeded for the first iteration
cc @fabriziopandini